src/06-Driver/14-Subscribing-to-Messages.purs

```diff
@@ -35,7 +35,7 @@ main =
 
       io.subscribe $ CR.consumer \raisedIntVal -> do
         liftEffect $ log $
-          "Consumer 1: Button was clicked and produced a random integer \
+          "Consumer 2: Button was clicked and produced a random integer \
```

Comment:
Both consumers had Consume 1 which was confusing..

Reply:
Ah, another copy and paste typo. Thanks for fixing that!